app/matplotlib.ipynb

```diff
@@ -339,18 +353,65 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 15,
-   "outputs": [],
    "source": [
-    "# write your code here\n"
+    "data = pd.read_csv(r'C:\\Users\\Alex\\py-matplotlib\\data\\customer_data.csv')\n",
+    "print(data.head())\n",
+    "sns.pairplot(data)\n",
+    "plt.show()\n",
```

Comment:
Task 9 is incomplete. Ensure that the scatter plot matrix is implemented as per the task requirements. You might need to add more code to visualize the relationships between different variables in the dataset.